flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
but I guess Flutter.....

Reply:
Yeah, aarch64 cross compiling for Flutter on x86_64 isn't really there yet from my experience. I want to support aarch64 but have had issues with trying to do so.